### PR TITLE
0.7.18b1

### PR DESCRIPTION
**Pull request by zhouaihui**

### CODE REVIEW

**Review by 6fj — DISMISSED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,12 +19,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+
+## [0.7.18b1] - 2023-1-30
+### Changed
+- add opiton whether exit on cross-silo sending.
```
Comment (Krout0n) [inline CHANGELOG.md:25]: typo?

**Review by anakinxc — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zhouaihui) [reply CHANGELOG.md]: Nice catch

File: docs/developer/design/programming_in_secretflow.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ cluster_config = {
     'parties': {
         'alice': {
             'address': '127.0.0.1:10001',
-        },
+        }
```
Comment (Krout0n) [inline docs/developer/design/programming_in_secretflow.md:79]:
```suggestion
        },
```

File: docs/developer/design/programming_in_secretflow.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ cluster_config = {
...
```
Comment (Krout0n) [inline docs/developer/design/programming_in_secretflow.md:98]:
```suggestion
        },
```

File: secretflow/ml/linear/ss_glm/model.py [hunk 4]
```diff
@@ -640,6 +657,8 @@ def predict(
 
             x : {FedNdarray, VDataFrame} of shape (n_samples, n_features)
                 Predict samples.
+            offset : {FedNdarray, VDataFrame} of shape (n_samples,)
```
Comment (Krout0n) [inline secretflow/ml/linear/ss_glm/model.py:660]:
```suggestion
            o : {FedNdarray, VDataFrame} of shape (n_samples,)
```

**Review by Krout0n — COMMENTED** [review]
Sorry for pointing out the details :bow:

File: secretflow/ml/linear/ss_glm/model.py (hunk 4 above)
Comment (zhouaihui) [reply secretflow/ml/linear/ss_glm/model.py]: Nice Catch!

**Review by 6fj — APPROVED** [review]
lgtm

**Pull request merged**